client/elements/suttaplex/sc-parallel-item.html

```diff
@@ -214,7 +214,8 @@
                       type: String,
                       statePath: 'siteLanguage',
                       observer: '_siteLanguageChanged'
-                  }
+                  },
+                  expansionData: Array
```

Comment:
I don't think we discussed this earlier, but a while ago we decided with Kuba that the

```
expansionData: {
    type: Array
}
```
syntax was superior to this one :). It's your call, but at the end it's about readability and keeping a uniform code style. If you take a look at the Google Polymer elements, I think they insert 2 blank lines between each property.